src/routes/(app)/_components/layout/notifications/notification-messages.ts

```diff
@@ -1,5 +1,6 @@
 export default Object.freeze<{
   [key: string]: string;
 }>({
-  COPY_EMAIL: 'O email foi copiado para o teu clipboard :)'
+  COPY_EMAIL: 'O email foi copiado para o teu clipboard :)',
+  INVALID_IMAGE: 'Imagem inválida! Por favor, seleciona outra.'
```

Comment:
I think this is more clear about the file type being the problem

```suggestion
   NOT_AN_IMAGE: 'O ficheiro não é uma imagem. Por favor, seleciona outro.'
```